src/UrqlClient.re

```diff
@@ -511,6 +511,32 @@ let query =
   |> Wonka.toPromise;
 };
 
+[@bs.send]
+external readQueryJs:
+  (
+    ~client: t,
+    ~query: string,
+    ~variables: Js.Json.t=?,
+    ~context: ClientTypes.partialOperationContextJs=?,
+    unit
+  ) =>
+  Js.Null.t('response) =
```

Comment:
I was not sure between `Js.Null.t` and `Js.Nullable.t`. I chose the former since it should reflect better the return type from the js side: https://github.com/FormidableLabs/urql/blob/master/packages/core/src/client.ts#L266

Reply:
✅ I think that makes sense. AFAIK, `Js.Nullable.t` is generally preferred to handle both `null` and `undefined` but since, as you point out, the JS side explicitly uses `null` I think this looks great.